tests/test_schemas.py

```diff
@@ -278,16 +278,26 @@ def test_parse_csv_model(
         assert "Attribute" in attr_rel_dictionary[attribute_key]["Relationships"]
 
 
-@pytest.mark.parametrize("data_model", ["example.model.jsonld"], ids=["jsonld"])
+@pytest.mark.parametrize(
+    "data_model",
+    [
+        "example.model.jsonld",
+        "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/refs/heads/develop/tests/data/example.model.jsonld",
```

Comment:
hmm should we point it to main instead? since eventually we are merging to main?